.travis.yml

```diff
@@ -76,6 +76,7 @@ before_install:
 
 install:
   - ccache -s
+  - git describe
```

Comment:
I did, was a bit worried that gh/travis were having caching / timeout issues and not running the code I thought it was running (turns out the problem is I had accidentally depended on dictionary ordering in 3.6).  I think this is a good idea to keep around so that we can verify _exactly_ what code travis runs for testing locally (it should be running on the merge into the target branch, when it re-sets what that merge is is not something I fully understand yet).